notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        display_name=DISPLAY_NAME,
</pre><p>This has error: <code>DISPLAY_NAME</code> is not defined. Please use <code>PIPELINE_DISPLAY_NAME</code>defined above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>done.</p>